src/middlewared/middlewared/plugins/pool_/pool.py

```diff
@@ -62,6 +63,10 @@ async def pool_normalize_info(self, pool_name):
         Common method for `pool.pool_extend` and `boot.get_state` returning a uniform
         data structure for its consumers.
         """
+        return await self.pool_normalize_info_impl(pool_name)
+
+    @private
```

Comment:
Why add this method? `pool_normalize_info` is private, so we can change its signature the way we want to accomodate the new `sed_cache` parameter.